README.md

```diff
@@ -1,66 +1,65 @@
-# Connect-Web
+<img src=".github/connect-logo.png" width="15%" />
 
-Connect-Web is a simple library to call remote procedures from a web browser. 
-Unlike REST, you get a type-safe client and never have to think about
-serialization again.
+# Connect for ECMAScript
 
-The procedures are defined in a [Protocol Buffer](https://developers.google.com/protocol-buffers)
-schema implemented by your backend, and Connect-Web generates the clients and
-related types to access the backend. The clients support two protocols:
-gRPC-web, and Connect's own protocol.
+Connect is a family of libraries for building and consuming APIs on 
+different languages and platforms. 
 
-The [Connect protocol](https://connect.build/docs/protocol/) is a simple,
-POST-only protocol that works over HTTP/1.1 or HTTP/2. It supports
-server-streaming methods just like gRPC-Web, but is easy to debug in the
-network inspector. Calling a Connect API is easy enough just with the fetch
-API. Try it with our live demo:
+With Connect, you define your schema first:
 
-```ts
-const res = await fetch("https://demo.connect.build/buf.connect.demo.eliza.v1.ElizaService/Say", {
-  method: "POST",
-  headers: {"content-type": "application/json"},
-  body: `{"sentence": "I feel happy."}`
-});
-const answer = await res.json();
-console.log(answer);
-// {sentence: 'When you feel happy, what do you do?'}
+```
+service ElizaService {
+  rpc Say(SayRequest) returns (SayResponse) {}
+}
 ```
 
-Using the client generated by Connect-Web, the same call becomes quite a bit 
-simpler:
+With the magic of code generation, this schema produces servers and clients:
```

Comment:
I think `And with the magic of code generation...` sounds a bit better maybe.

Reply:
Updated